src/main/java/com/example/CinemaRoom/service/StatisticsService.java

```diff
@@ -1,28 +1,48 @@
 package com.example.CinemaRoom.service;
 
+import com.example.CinemaRoom.model.Seat;
 import com.example.CinemaRoom.model.Statistics;
+import com.example.CinemaRoom.model.Ticket;
+import com.example.CinemaRoom.repository.SeatRepository;
+import com.example.CinemaRoom.repository.TicketRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.util.ArrayList;
+import java.util.List;
 
+@Service
 public class StatisticsService {
 
+    private SeatRepository seatRepository;
+    private TicketRepository ticketRepository;
     private Statistics statistics;
 
-    public StatisticsService(int available) {
-        this.statistics = new Statistics(available);
+    @Autowired
+    public void CinemaRoomApplication(SeatRepository seatRepository,TicketRepository ticketRepository) {
```

Comment:
Shouldn't this be a constructor?

```suggestion
    public StatisticsService(SeatRepository seatRepository,TicketRepository ticketRepository) {
```